### PR TITLE
Sync entitlement secret

### PR DESCRIPTION
**Pull request by stuartwdouglas**

### CODE REVIEW

**Review by gbenhaim — APPROVED** [review]

File: task/buildah-remote/0.1/buildah-remote.yaml [hunk 1]
```diff
@@ -189,6 +189,8 @@ spec:
       fi
 
       rsync -ra $(workspaces.source.path)/ "$SSH_HOST:$BUILD_DIR/workspaces/source/"
+      rsync -ra /entitlement/ "$SSH_HOST:$BUILD_DIR/volumes/etc-pki-entitlement/"
+      rsync -ra /mnt/trusted-ca/ "$SSH_HOST:$BUILD_DIR/volumes/trusted-ca/"
```
Comment (gbenhaim) [inline task/buildah-remote/0.1/buildah-remote.yaml:193]: will it still work if `/mnt/trusted-ca` doesn't exist? mounting the configmap to that directory is optional and only happens with the configmap with the ca bundle is configured in the namespace.

Comment (stuartwdouglas) [reply task/buildah-remote/0.1/buildah-remote.yaml:193]: Sorry, I forgot to respond to this. I tested it locally and it seemed to work fine.

Comment (gbenhaim) [reply task/buildah-remote/0.1/buildah-remote.yaml:193]: thanks for checking

**Review by gbenhaim — APPROVED** [review]

**Comment by gbenhaim** [comment]
@stuartwdouglas why did you close this one? is there another pr for making those changes?

**Pull request closed without merging**